.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\ProgramData\\Anaconda3\\python.exe"
+}
```

Comment:
Add `.vscode` to your .gitignore file and `git rm` this file. Generally, you don't want to track system files, passwords, certificates etc. in github, especially in a public repository!